worldedit-core/src/main/java/com/sk89q/worldedit/math/MathUtils.java

```diff
@@ -105,4 +105,20 @@ public static double dSin(double degrees) {
         return Math.sin(Math.toRadians(degrees));
     }
 
+    /**
+     * Returns the rounded double of the given value. This is needed because
+     * {@code Math.round(value)} implementation rounds a.5, where a is an integer,
+     * regardless of sign, to the closest greater integer.
+     * ("Round up")
+     *
+     * In other words, {@code Math.round(-3.5)}, as an example, rounds -3.5 to -3
+     * instead of the "typical" -4. The following function has the "typical"
+     * behavior for both positive and negative values.
+     *
+     * @param value the value
+     * @return the rounded value
+     */
+    public static double round(double value) {
```

Comment:
except those are all different things and none are standard, especially when it comes to the applications in negatives here.
https://docs.oracle.com/javase/7/docs/api/java/math/RoundingMode.html